package.json

```diff
@@ -113,6 +113,7 @@
     "redux": "^4.0.5",
     "redux-mock-store": "^1.5.4",
     "redux-testkit": "^1.0.6",
+    "@types/antd": "^1.0.0",
```

Comment:
antd official site says, this is not required actually it says we should not add it: https://ant.design/docs/react/use-in-typescript.